src/components/network/network-map-tab.tsx

```diff
@@ -606,21 +601,21 @@ export const NetworkMapTab = ({
         setGeoData(undefined);
         geoDataRef.current = null;
 
-        // @ts-expect-error TODO: manage rootNodeId undefined case
-        const substationPositionsDone = fetchSubstationPositions(studyUuid, rootNodeId, currentRootNetworkUuid).then(
-            (data) => {
-                console.info(`Received substations of study '${studyUuid}'...`);
-                const newGeoData = new GeoData(new Map(), geoDataRef.current?.linePositionsById || new Map());
-                newGeoData.setSubstationPositions(data);
-                setGeoData(newGeoData);
-                geoDataRef.current = newGeoData;
-            }
-        );
+        const substationPositionsDone = fetchSubstationPositions(
+            studyUuid,
+            rootNodeId as UUID,
```

Comment:
This is hiding the fact it could be undefined

Reply:
rollback